zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/NoteManager.java

```diff
@@ -247,7 +247,7 @@ public void moveNote(String noteId,
     this.notebookRepo.move(noteId, notePath, newNotePath, subject);
 
     // Update path of the note
-    if (!StringUtils.equalsIgnoreCase(notePath, newNotePath)) {
+    if (!notePath.equals(newNotePath)) {
```

Comment:
`StringUtils.equals(x, y)` is null safe and should be preferred.

Reply:
Git: case sensitive
Hadoop: case sensitive
S3: case sensitive
Azure: case sensitive
Google Cloud Storage: case sensitive
Aliyun OSS: case sensitive
MongoDB: case sensitive
Github: case sensitive

It seems we are safe.

Reply:
awesome